test/enhance/test_normalize.py

```diff
@@ -217,3 +218,61 @@ def test_single_value(self):
         expected = (data * std) + mean
 
         assert_allclose(kornia.denormalize(data, mean, std), expected)
+
+
+class TestNormalizeMinMax(BaseTester):
+    def test_smoke(self, device, dtype):
+        x = torch.ones(1, 1, 1, 1, device=device, dtype=dtype)
+        assert kornia.normalize_min_max(x) is not None
+        assert kornia.enhance.normalize_min_max(x) is not None
+
+    def test_exception(self, device, dtype):
+        with pytest.raises(TypeError):
+            assert kornia.normalize_min_max(0.)
+            assert kornia.normalize_min_max(
+                torch.ones(1, 1, device=device, dtype=dtype), '')
+            assert kornia.normalize_min_max(
+                torch.ones(1, 1, device=device, dtype=dtype), 2., '')
```

Comment:
Wouldn't the first raised TypeError already make it pass?

Reply:
@FGeri I will check on that. I'm learning about pytest.raises :)

Reply:
@FGeri please make suggestions on that to improve testing

Reply:
@edgarriba `with pytest.raises()` exits the context after the first error is raised. Thus, if the first call fails, the latter two will not be hit at all. You need to wrap every call that should fail with a `TypeError` within a `with pytest.raises(TypeError):`.